src/lostcity/engine/GameMap.ts

```diff
@@ -202,13 +235,13 @@ export default class GameMap {
             for (let index: number = 0; index < count; index++) {
                 const id: number = packet.g2();
                 const count: number = packet.g1();
-                if (!Environment.NODE_MEMBERS && !this.isFreeToPlay(absoluteX, absoluteZ)) {
+                if (!this.members && !this.isFreeToPlay(absoluteX, absoluteZ)) {
                     continue;
                 }
                 const objType: ObjType = ObjType.get(id);
                 const obj: Obj = new Obj(level, absoluteX, absoluteZ, EntityLifeCycle.RESPAWN, objType.id, count);
-                if (objType.members && Environment.NODE_MEMBERS || !objType.members) {
-                    zoneMap.zone(obj.x, obj.z, obj.level).addStaticObj(obj);
+                if (objType.members && this.members || !objType.members) {
```

Comment:
just FYI you can get rid of the double-clause here with:

```suggestion
                if (!objType.members || this.members) {
```

Don't feel you have to change it for this PR though  just letting you know